docs/pages/reference/terraform-provider/data-sources/workload_identity.mdx

```diff
@@ -105,6 +105,7 @@ Optional:
 Optional:
 
 - `dns_sans` (List of String) The DNS Subject Alternative Names (SANs) that should be included in an X509-SVID issued using this WorkloadIdentity.  Each entry in this list supports templating using attributes.
+- `maximum_ttl` (String) Control the maximum TTL of X509-SVIDs issued using this WorkloadIdentity.  If a X509-SVID is requested with a TTL greater than this value, then the returned X509-SVID will have a TTL of this value.  Defaults to 24 hours. The maximum this value can be set to is 14 days.
```

Comment:
This file should have two changes 🤔 
One for `spec.spiffe.x509` (this one) and another one for the JWT

Reply:
hm

Reply:
Just a guess: maybe you added the X509 max ttl field first and generated the docs
Meanwhile, you also added the JWT max ttl but didn't generate the docs